theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveEditorPopover.tsx

```diff
@@ -181,10 +194,6 @@ const CurveEditorPopover: React.FC<IProps> = (props) => {
     }
   }
 
-  // In auto mode, the text input field is continually updated to
-  // a CSS cubic bezier args string to reflect the state of the curve;
-  // in user mode, the text input field does not update when the curve
-  // changes so that the user's search is preserved.
   const [textInputMode, setTextInputMode] = useState<TextInputMode>(
     TextInputMode.init,
   )
```

Comment:
Moved so we can get this relevant info on focus of the actual values like hovering over `TextInputMode.auto` in the context of `setTextInputMode(TextInputMode.auto)`